docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,74 @@
+# Migrate from another CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea since
+Antrea v1.15.0.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work. Please create an issue if you run into problems during the
+migration to Antrea. During the migration process, no Kubernetes resources
+should be created or deleted, otherwise the migration process might fail or
+some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up the old CNI.
+2. Install Antrea in the cluster.
+3. Deploy Antrea migrator.
+
+## Clean up the old CNI
+
+The cleanup process varies across CNIs, typically you should remove
+the DaemonSet, Deployment, and CRDs of the old CNI from the cluster.
+For example, if you used `kubectl apply -f <CNI_MANIFEST>` to install
+the old CNI, you could then use `kubectl delete -f <CNI_MANIFEST>` to
+uninstall it.
+
+## Install Antrea
+
+The second step is to install Antrea in the cluster. You can follow the
+[installation guide](https://github.com/antrea-io/antrea/blob/main/docs/getting-started.md)
+to install Antrea. The following is an example of installing Antrea v1.14.1:
+
+```bash
+$ kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.14.1/antrea.yml
```

Comment:
nit: remove `$` when it's pure commands so it's easier to copy via a click. `$` is meaningful when used to differentiate command and output.

Reply:
Removed the dollar symbol.